tasks/phantom-dc.rake

```diff
@@ -24,15 +25,33 @@ namespace :'phantom-dc' do
         cm_id, cm_args = DelayedJobHelper::congress_member_id_and_args_from_handler(job.handler)
         unless cm_args[1] == "rake" and args[:job_id].nil?
           cm = CongressMember::retrieve_cached(cm_hash, cm_id)
+
           if regex.nil? or regex.match(cm.bioguide_id)
-            if retrieve_captchad_cached(captcha_hash, cm.id)
+            if cm.has_google_recaptcha?
+              recaptcha_jobs.push job
```

Comment:
You probably want to do something like building a `recaptcha_hash` as we do for the `captcha_hash` above.

The reason we do this is to avoid an unnecessary database lookup of the member of congress associated with the current job, which happens for every job within this loop.  If there are some number of thousands of jobs, this results in the sieving step into `recaptcha_jobs`, `captcha_jobs`, and `noncaptcha_jobs` here to take much longer.

If you build a hash of the recaptcha'd members from the outset, there is only one lookup per MoC.  You could even build it into the `build_captcha_hash` method, to make line 22 above read:

```ruby
captcha_hash, recaptcha_hash = build_captcha_hash
```